index.js

```diff
@@ -136,6 +138,8 @@ HomeAssistantPlatform.prototype = {
           accessory = new HomeAssistantSwitch(that.log, entity, that, 'scene')
         }else if (entity_type == 'media_player' && entity.attributes && entity.attributes.supported_media_commands){
           accessory = new HomeAssistantMediaPlayer(that.log, entity, that)
+        }else if (entity_type == 'thermostat'){
```

Comment:
Isn't this climate now?

Please be sure you're on the most recent version of Home Assistant while developing this.